etc/profile-m-z/mirage.profile

```diff
@@ -11,6 +11,8 @@ noblacklist ${HOME}/.config/mirage
 noblacklist ${HOME}/.local/share/mirage
 noblacklist /sbin
```

Comment:
Site note: This break for systems with a unified file-system-hirachy w/o `nobalcklist /usr/sbin`.

Reply:
I didn't introduce it; it was by reinerh for the `matrix-mirage`  Debian package https://github.com/netblue30/firejail/pull/3854/files#diff-a1f8494c6f1dd528c0cc8bba9fd3d8739afcd1e3baef0e1f112fa4d91c6d6b08R12; maybe it can be moved to that profile or add a note here?